news_items/004_freq_testing_paper/index.html

```diff
@@ -0,0 +1,54 @@
+<div class="d-flex justify-content-center">
+  <div class="card mb-4" style="width: 80%;">
+    <div class="card-body bg-light">
+      Morphew, J. W., Silva, M., Herman, G., and West, M., <b>"Frequent mastery testing with second-chance exams leads to enhanced student learning in undergraduate engineering"</b>, <i>Applied Cognitive Psychology</i> 34(1), 2020. DOI: <a href="https://doi.org/10.1002/acp.3605">10.1002/acp.3605</a>. Full-text link: <a href="http://lagrange.mechse.illinois.edu/pubs/MoSiHeWe2020/MoSiHeWe2020.pdf">PDF</a>.
+    </div>
+  </div>
+</div>
+
+<p>
+  In this recently published paper, <a href="https://engineering.purdue.edu/ENE/People/ptProfile?resource_id=222158">Jason Morphew</a> and co-authors showed that frequent short PrairieLearn exams in the CBTF (Computer-Based Testing Facility) substantially increased student learning in TAM 251 ("Introductory Solid Mechanics") at the University of Illinois. Most impressively, these learning gains were on an identical pen-and-paper final exam and there were improvements at all grade levels.
```

Comment:
Maybe mention this consisted of 13 multiple-choice questions and three free-response questions in the given–find–solution format?

Also, was the rubric/graders were held constant in the 3 free response questions. Plus, how well does the 16 question format final across years work?

Reply:
I'll mention the types of questions in the final -- I agree that's useful context.

The rubric was constant, the graders were not. The paper did not include data on uniformity of the final across years, so I won't include any here. I'm trying to keep this type of post to summarizing what's in the paper, not adding more information.